startup_gateway/chat/consumers.py

```
@@ -104,19 +122,128 @@ async def handle_send(self, data):
+            )
+
+            await self.channel_layer.group_send(
+                f'conversation_{conversation_id}',
+                {
+                    'type': 'read_receipt',
+                    'user_id': self.user.id,
+                    'conversation_id': conversation_id,
+                    'count': count,
+                }
+            )
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)